lumeer-core/src/main/java/io/lumeer/core/facade/SuggestionFacade.java

```diff
@@ -112,8 +113,11 @@ private boolean shouldSuggestViews(SuggestionQuery suggestionQuery) {
    }
 
    private List<View> suggestViews(SuggestionQuery suggestionQuery, int limit) {
-      SearchSuggestionQuery searchSuggestionQuery = createSuggestionQuery(suggestionQuery, limit);
-      return viewDao.getViews(searchSuggestionQuery, isManager());
+      var distance = LevenshteinDistance.getDefaultInstance();
```

Comment:
I don't know how heavy this object is and how it works internally but I just wonder if it cannot be stored as a static attribute so it doesn't have to be created every time this method is called.

Reply:
`getDefaultInstance()` returns static instance under the hood.